src/libs/svg/lv_svg_parser.c

```diff
@@ -99,6 +99,7 @@ static const struct _lv_svg_attr_map {
     {"points", 6, LV_SVG_ATTR_POINTS},
     {"d", 1, LV_SVG_ATTR_D},
     {"pathLength", 10, LV_SVG_ATTR_PATH_LENGTH},
+    {"style", 5, LV_SVG_ATTR_STYLE},
```

Comment:
The array element index does not match the enumeration value, which will result in inconsistent output values ​​when debugging the SVG DOM tree. 
See：https://github.com/lvgl/lvgl/pull/9423/files#diff-9751c79d8395a0caf716fc1288d1793bfa39b58a39e3975464ebbe2bfeeff23dL2345:~:text=void%20_lv_svg_dump_tree(,%7B